picker/leastloaded.go

```diff
@@ -86,23 +86,20 @@ type leastLoadedConnHeap []*leastLoadedConnItem
 type leastLoadedConnItem struct {
 	conn     conn.Conn
 	load     uint64
-	tiebreak uint64
+	tieBreak uint64
```

Comment:
We have `nextTieBreak` with a capital B elsewhere, so I capitalized it here for consistency that this represents two words.